File: FinanceApp.Shared/Dto/FGTS/CreateOrUpdateFGTS.cs
using FinanceApp.Shared.Dto.Base;

namespace FinanceApp.Shared.Dto.FGTS
{
    public class CreateOrUpdateFGTS : UpdateDto
    {

        public double CurrentBalance { get; set; }
        public double MonthlyGrossIncome { get; set; }
        public bool AnniversaryWithdraw { get; set; }

    }
}

File: FlutterApp/finance_app/lib/models/fgts/create_or_update_fgts.dart
class CreateOrUpdateFGTS {
  double monthlyGrossIncome;
  double currentBalance;
  bool anniversaryWithdraw;
  int monthAniversaryWithdraw;

  CreateOrUpdateFGTS({
    required this.currentBalance,
    required this.anniversaryWithdraw,
    required this.monthAniversaryWithdraw,
    required this.monthlyGrossIncome,
  });

  Map<String, dynamic> toJson() => {
        'currentBalance': currentBalance,
        'anniversaryWithdraw': anniversaryWithdraw,
        'monthAniversaryWithdraw': monthAniversaryWithdraw,
        'monthlyGrossIncome': monthlyGrossIncome,
      };
}
